worker.py

```diff
@@ -198,8 +197,7 @@ def run(self):
             # Assigning job to a new VM
             else:
                 self.log.debug("Assigning job to a new VM")
-                self.job.vm.id = self.job.id
-                self.job.updateRemote()
+                self.job.makeVM(self.job.id)
```

Comment:
since we're setting `self.vm` instead of `self.vm.id` in `makeVM`, wouldn't this cause VM be set to a string here?

Reply:
You're right, it should be `self.job`